Features/NFT/Sources/Scenes/NFTScene.swift

```diff
@@ -20,14 +20,20 @@ public struct NFTScene: View {
         ]
     }
     
+    @Binding var isPresentingSelectType: SelectAssetType?
```

Comment:
```suggestion
    @Binding var isPresentingSelectType: SelectAssetType?
```

No need to include it here, navigation stack should handle it. toolbar should just call `model.selectAssetType`. and then communicate this to the navigation stack